### ISSUE TITLE
Mood icons rarely show up

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1232]
In this game it feels you have very little control of what your creatures are doing. Part of this is that you always see creatures wander off for no reason.
Mood icons would be perfect for this, but for some reason you only see those occasionally.

Whenever a creature is leaving an assigned room, he should show the reason why with the mood icon.

I've attached a save from git20160725 to see cultists walking out of the training room. Only one shows the 'sleep' mood icon, the others don't.
[2016-07-27_001228-SK-Angel.zip](https://github.com/OpenDungeons/OpenDungeons/files/385006/2016-07-27_001228-SK-Angel.zip)

**Comment by akien-mga** [comment]
As far as I know the mood icons should show on hover or when you press <kbd>Alt</kbd> IIRC. When a creature has several mood affections, they should be shown one after the other in a rotation. If it doesn't, it might be a regression.

**Comment by Loobinex** [comment]
I did press Alt as well, and no mood Icons showed up.

**Comment by hwoarangmy** [comment]
No, mood icons are always displayed. Only the health icon is hidden and appears when pressing alt.

However, ATM, no mood icon says why a creature stops working as it may stop before getting too hungry or sleepy.

**Comment by Loobinex** [comment]
Well, that needs to be tweaked then. The whole point of the mood icons is to give visual feedback to the player on what a creature is going to do. In other words, if a creature is hungry enough to stop training, he is hungry enough to show the mood icon.
